### PR TITLE
Remove usages of StoredFileNode and FileNode [OSF-7854]

### PR DESCRIPTION
**Pull request by erinspace**
## Purpose

Currently, StoredFileNode and FileNode are aliases of BaseFileNode, and they only exist for backwards compatibility.

## Changes

- Remove aliases of `StoredFileNode` and `FileNode` from new OSF models
- Switch usages in tests, views, serializers, etc to use `BaseFileNode`

## Side effects

none anticipated

## Ticket
https://openscience.atlassian.net/browse/OSF-7854

### CODE REVIEW

File: addons/base/views.py [hunk 1]
```diff
@@ -439,7 +439,7 @@ def create_waterbutler_log(payload, **kwargs):
 
 @file_signals.file_updated.connect
 def addon_delete_file_node(self, node, user, event_type, payload):
-    """ Get addon StoredFileNode(s), move it into the TrashedFileNode collection
+    """ Get addon BsaeFileNode(s), move it into the TrashedFileNode collection
```
Comment (alexschiller) [inline addons/base/views.py]: stupid driveby comment-- typo :)

Comment (erinspace) [inline addons/base/views.py]: thank yoooooou
